state/dataTrieValue/dataTrieValue.pb.go

```diff
@@ -0,0 +1,499 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: dataTrieValue.proto
+
+package dataTrieValue
+
+import (
+	bytes "bytes"
+	fmt "fmt"
+	_ "github.com/gogo/protobuf/gogoproto"
+	proto "github.com/gogo/protobuf/proto"
+	io "io"
+	math "math"
+	math_bits "math/bits"
+	reflect "reflect"
+	strings "strings"
+)
+
+// Reference imports to suppress errors if they are not otherwise used.
+var _ = proto.Marshal
+var _ = fmt.Errorf
+var _ = math.Inf
+
+// This is a compile-time assertion to ensure that this generated file
+// is compatible with the proto package it is being compiled against.
+// A compilation error at this line likely means your copy of the
+// proto package needs to be updated.
+const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
+
+type TrieLeafData struct {
```

Comment:
`TrieLeafData` is not being used yet, it will be used when returning parsing the leafs, or elsewhere? because in the interface it returns core.KeyValueHolder right now

Reply:
Yes, it will be used in the next PR, in which a real `trieLeafParser` will be implemented.